{"uid":"a908d69909da8ab1","name":"test_freeze_db_process[/usr/bin/patroni]","fullName":"tests.integration.ha_tests.test_self_healing_1#test_freeze_db_process","historyId":"7f29c782589fd64137fad71896320c17","time":{"start":1758073838397,"stop":1758073838397,"duration":0},"status":"skipped","statusMessage":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 843, in pytest_runtest_setup\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 895, in pytest_runtest_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 164, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1674, in setup\n    self._request._fillfixtures()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 719, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 548, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 639, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1127, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_operator/plugin.py\", line 250, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1758072642515,"stop":1758072642515,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1758072642515,"stop":1758072642515,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1758072642515,"stop":1758072642782,"duration":267},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"primary_start_timeout","time":{"start":1758073053998,"stop":1758073060164,"duration":6166},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1758073838397,"stop":1758073838397,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_operator/plugin.py\", line 250, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"primary_start_timeout::finalizer","time":{"start":1758073838420,"stop":1758073840500,"duration":2080},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1758073840501,"stop":1758073841256,"duration":755},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1758073841258,"stop":1758073841259,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1758073841259,"stop":1758073841259,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1758073841259,"stop":1758073841259,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing_1"},{"name":"host","value":"runnervmmq7i7"},{"name":"thread","value":"20736-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing_1"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"process","value":"'/usr/bin/patroni'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"f1c4bc4115bfffc7","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":14,"broken":0,"skipped":44,"passed":137,"unknown":12,"total":207},"items":[{"uid":"55c2cb07e31837b1","reportUrl":"../4528//#testresult/55c2cb07e31837b1","status":"passed","time":{"start":1757987761062,"stop":1757987854675,"duration":93613}},{"uid":"f77e1c231755771d","reportUrl":"../4524//#testresult/f77e1c231755771d","status":"passed","time":{"start":1757901219018,"stop":1757901330439,"duration":111421}},{"uid":"f278d93ba042ad6a","reportUrl":"../4521//#testresult/f278d93ba042ad6a","status":"unknown","time":{}},{"uid":"f67b3e9e236342af","reportUrl":"../4520//#testresult/f67b3e9e236342af","status":"unknown","time":{}},{"uid":"2e39e20fd126d6d7","reportUrl":"../4515//#testresult/2e39e20fd126d6d7","status":"unknown","time":{}},{"uid":"5e947acb72e5a717","reportUrl":"../4509//#testresult/5e947acb72e5a717","status":"unknown","time":{}},{"uid":"641f00709f1dd298","reportUrl":"../4500//#testresult/641f00709f1dd298","status":"unknown","time":{}},{"uid":"dc50c9feb15388df","reportUrl":"../4489//#testresult/dc50c9feb15388df","status":"unknown","time":{}},{"uid":"41009f8e3090d50","reportUrl":"../4488//#testresult/41009f8e3090d50","status":"passed","time":{"start":1757296329858,"stop":1757296410230,"duration":80372}},{"uid":"8be7b9ae5f6c556f","reportUrl":"../4486//#testresult/8be7b9ae5f6c556f","status":"failed","statusDetails":"AssertionError: testing.postgresql-k8s-1: writes to the db were missed.","time":{"start":1757123748846,"stop":1757123844915,"duration":96069}},{"uid":"4c8bdcc46d1de2c2","reportUrl":"../4479//#testresult/4c8bdcc46d1de2c2","status":"skipped","statusDetails":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1756950680868,"stop":1756950680868,"duration":0}},{"uid":"ced5218f127bfd9","reportUrl":"../4476//#testresult/ced5218f127bfd9","status":"passed","time":{"start":1756864509517,"stop":1756864589835,"duration":80318}},{"uid":"6e4fa42bea99fbc7","reportUrl":"../4467//#testresult/6e4fa42bea99fbc7","status":"unknown","time":{}},{"uid":"dbf274029d773782","reportUrl":"../4466//#testresult/dbf274029d773782","status":"passed","time":{"start":1756691650142,"stop":1756691742227,"duration":92085}},{"uid":"31e92053c4ce589f","reportUrl":"../4465//#testresult/31e92053c4ce589f","status":"skipped","statusDetails":"XFAIL aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1756605254736,"stop":1756605254736,"duration":0}},{"uid":"5651c8e5bca7be65","reportUrl":"../4464//#testresult/5651c8e5bca7be65","status":"unknown","time":{}},{"uid":"fbbfb95a9ea40029","reportUrl":"../4455//#testresult/fbbfb95a9ea40029","status":"unknown","time":{}},{"uid":"4aeddf632d530d05","reportUrl":"../4452//#testresult/4aeddf632d530d05","status":"failed","statusDetails":"AssertionError: testing.postgresql-k8s-0: writes to the db were missed.","time":{"start":1756346164195,"stop":1756346253309,"duration":89114}},{"uid":"17633c53c16237c9","reportUrl":"../4451//#testresult/17633c53c16237c9","status":"passed","time":{"start":1756259632590,"stop":1756259712882,"duration":80292}},{"uid":"49593a767efcf430","reportUrl":"../4440//#testresult/49593a767efcf430","status":"passed","time":{"start":1756174962442,"stop":1756175059768,"duration":97326}}]},"tags":["asyncio","abort_on_fail"]},"source":"a908d69909da8ab1.json","parameterValues":["'/usr/bin/patroni'"]}